simulator/infra/ports/module.h

```diff
@@ -9,9 +9,11 @@
  
 #include <infra/log.h>
 #include <infra/ports/ports.h>
-
 #include <unordered_set>
 
+//Forward declarations
+#include <boost/property_tree/ptree_fwd.hpp>
```

Comment:
Please follow our guidelines on includes order: internal headers, Boost headers, C++ headers (https://github.com/MIPT-ILab/mipt-mips/wiki/Code-style-guidelines#order-of-includes)